src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -772,6 +784,22 @@ private void buildExecParams(TDSWriter tdsWriter) throws SQLServerException {
         assert hasPreparedStatementHandle();
         tdsWriter.writeRPCInt(null, getPreparedStatementHandle(), false);
     }
+    
+    private void buildRPCExecParams(TDSWriter tdsWriter) throws SQLServerException {
+        if (getStatementLogger().isLoggable(java.util.logging.Level.FINE))
+            getStatementLogger().fine(toString() + ": calling sp_execute: PROC" + ", SQL:" + preparedSQL);
```

Comment:
This should not say "calling sp_execute" since we are doing a direct SQLRPC call to the specific procedure.

Reply:
Yes! good catch.